packages/usa-modal/src/test/modal-nested-form.spec.js

```diff
@@ -0,0 +1,161 @@
+      overlay = body.querySelector(".usa-modal-overlay");
+      openButton1 = openButton1Selector();
+      openButton2 = openButton2Selector();
+
+      comboList = comboListSelector();
+      CBToggle = body.querySelector(".usa-combo-box__toggle-list");
+
+    });
+
+    afterEach(() => {
+      modal.off(containerSelector());
+      comboBox.off(containerSelector());
+      body.innerHTML = "";
+      body.className = "";
+    });
+
+    describe("Builds the modal HTML", () => {
+      it("creates new parent elements", () => {
+        assert.ok(modalWindow, "creates inner div");
+        assert.ok(overlay, "creates the overlay");
+        assert.ok(modalWrapper, "creates the outer div");
+      });
+
+      it('adds role="dialog" to modal parent', () => {
+        assert.strictEqual(modalWrapper.getAttribute("role"), "dialog");
+      });
+
+      it("moves aria-labelledby, aria-describedby, and id to the parent", () => {
+        assert.strictEqual(modalWindow.hasAttribute("aria-describedby"), false);
+        assert.strictEqual(modalWindow.hasAttribute("aria-labelledby"), false);
+        assert.strictEqual(modalWindow.hasAttribute("id"), false);
+        assert.strictEqual(modalWrapper.hasAttribute("aria-describedby"), true);
+        assert.strictEqual(modalWrapper.hasAttribute("aria-labelledby"), true);
+        assert.strictEqual(modalWrapper.getAttribute("id"), "modal");
+      });
+
+      it('sets tabindex="-1" to the modal window', () => {
+        assert.strictEqual(modalWindow.getAttribute("tabindex"), "-1");
+      });
+
+      it("moves the modal to the bottom of the DOM", () => {
+        assert.strictEqual(
+          body.lastElementChild.classList.contains("usa-modal-wrapper"),
+          true
+        );
+      });
+
+      it('adds role="button" to any <a> opener, but not <button>', () => {
+        assert.strictEqual(openButton1.getAttribute("role"), "button");
+        assert.strictEqual(openButton2.hasAttribute("role"), false);
+      });
+
+      it("adds aria-controls to each opener", () => {
+        assert.strictEqual(openButton1.getAttribute("aria-controls"), "modal");
+        assert.strictEqual(openButton2.getAttribute("aria-controls"), "modal");
+      });
+    });
+
+    describe("When opened", () => {
+      beforeEach(() => {
+        openButton1.click();
+      });
+
+      it("makes the modal visible", () => {
+        assert.strictEqual(isVisible(modalWrapper), true);
+      });
+
+      it("focuses the modal window when opened", () => {
+        assert.strictEqual(document.activeElement, modalWindow);
+      });
+
+      it("makes all other page content invisible to screen readers", () => {
+        const activeContent = document.querySelectorAll(
+          "body > :not([aria-hidden])"
+        );
+
+        assert.strictEqual(activeContent.length, 1);
+        assert.strictEqual(activeContent[0], modalWrapper);
+      });
+
+      it("should show the list by clicking the toggle button", () => {
```

Comment:
Wondering if this could be more clear. Maybe something like "Displays combobox list when clicked"?

Reply:
Oh yeah I copied this from the combo box test, I'll update this to be more clear 😁